repositories.bzl

```diff
@@ -35,29 +38,51 @@ VERSIONS = {
         "strip_prefix": "zlib-1.2.11",
         "build_file": "@rules_proto_grpc//third_party:BUILD.bazel.zlib",
     },
-
-    # Misc
+    "rules_python": {
+        "type": "github",
+        "org": "bazelbuild",
+        "repo": "rules_python",
+        "ref": "94677401bc56ed5d756f50b441a6a5c7f735a6d4",
+        "sha256": "acbd018f11355ead06b250b352e59824fbb9e77f4874d250d230138231182c1c",
+    },
+    "build_bazel_rules_swift": {
+        "type": "github",
+        "org": "bazelbuild",
+        "repo": "rules_swift",
+        "ref": "0.13.0",
+        "sha256": "617e568aa8263c454f63362f5ab837038da710d646510b8f4a6760ff6361f714",
+    },
+    "build_bazel_apple_support": {
+        "type": "github",
+        "org": "bazelbuild",
+        "repo": "apple_support",
+        "ref": "0.7.2",
+        "sha256": "519a3bc32132f7b5780e82c2fc6ad2a78d4b28b81561e6fd7b7e0b14ea110074",
+    },
     "bazel_skylib": {
         "type": "github",
         "org": "bazelbuild",
         "repo": "bazel-skylib",
-        "ref": "be3b1fc838386bdbea39d9750ea4411294870575", # Apr 13, 2019
-        "sha256": "6128dd2af9830430e0ae404cb6fdce754fb80ed88942e1a0865a7f376bb68c4e",
+        "ref": "1.0.2",
+        "sha256": "e5d90f0ec952883d56747b7604e2a15ee36e288bb556c3d0ed33e818a4d971f2",
     },
+
+
+    # Misc
     "com_github_bazelbuild_buildtools": {
         "type": "github",
         "org": "bazelbuild",
         "repo": "buildtools",
-        "ref": "6415663945d3248207da955aafa1fa2af1a0f2ed",
-        "sha256": "d1e28237d1f4c2255c504246b4f3fd36f74d590f2974491b4399a84c58b495ed",
+        "ref": "0.28.0",
+        "sha256": "d1e28237d1a4c2255c504246b4f3fd36f74d590f2974491b4399a84c58b495ed",
```

Comment:
bazel reports `5ec71602e9b458b01717fab1d37492154c1c12ea83f881c745dbd88e9b2098d8` for this one

Reply:
Good spot, fixed, thanks